app/presenters/dashboard_widget_presenter.rb

```diff
@@ -9,7 +9,8 @@ class DashboardWidgetPresenter
   def initialize(name, params = {}, variables = {})
     @name = name
     @params = params.respond_to?(:permit!) ? params.dup.permit! : params
-    @params = @params.symbolize_keys.with_indifferent_access
+    # symbolize and with_indifferent_access deprecated
```

Comment:
perhaps leave this as a pull request comment and remove from code. Is it needed to be in the code?